pkg/master/master.go

```diff
@@ -265,7 +265,7 @@ func (cfg *Config) Complete(informers informers.SharedInformerFactory) Completed
 		// We should probably allow this for clouds that don't require NodePort to do load-balancing (GCE)
 		// but then that breaks the strict nestedness of ServiceType.
 		// Review post-v1
-		c.ExtraConfig.ServiceNodePortRange = options.DefaultServiceNodePortRange
+		c.ExtraConfig.ServiceNodePortRange = kubeoptions.DefaultServiceNodePortRange
```

Comment:
Uggggggh.  All of this default-setting in multiple places is horrific.  It's been a long time since I touched this, but could we find a way to force callers to pass valid values?

Reply:
@thockin yes we can, in a follow up PR. Want to keep it as simple as i can.